lib/tanker/identity.rb

```diff
@@ -118,7 +118,7 @@ def self.get_public_identity(serialized_identity)
       public_keys.each { |key| public_identity[key] = identity.fetch(key) }
 
       if identity['target'] == 'email'
-        identity['target'] = 'hashed_email'
+        public_identity['target'] = 'hashed_email'
```

Comment:
can squash this commit

Reply:
It's a separate bug that was already there, so if I squash with the next breaking change you lose the information about the bug in the other big commit =)